app/conversations/utils.py

```diff
@@ -0,0 +1,35 @@
+from app.models import Conversations, Users
+from app import db
+
+
+def build_single_conversation_response(conversation_uuid):
+    """
+    Deal with database interactions to provide response for get single conversation request.

```

Comment:
Can you add the inputs and outputs to this function description? We want it to be a standard that every function has a description of what the function does as well as the input parameters listed with their description and description of everything returned by the function. Mimic the format used in other newer functions in the app. The older functions might not use the format but we are trying to switch to have all the functions use the documentation format.

Reply:
Yup.